tests/test_producer.py

```
@@ -0,0 +1,269 @@
+            put_bandwidth_limit_per_shard=1,  # Very low bandwidth limit
+        ) as producer:
+            # Add several records quickly
+            for i in range(5):
+                await producer.put({"message": f"test-{i}"})
+
+            await producer.flush()
+
+    @pytest.mark.asyncio
+    async def test_producer_error_handling(self, endpoint_url):
+        """Test producer error handling with invalid stream."""
+        with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)